### DISCUSSION TITLE
Github is locking my account by 2FA, but I have no public contributions

### DISCUSSION THREAD

**Discussion by marcinnformula** [community/community#77914] [Other Feature Feedback, Questions, & Ideas]
My account is going to be locked because of some criteria, but I have no public contributions. This is account for private and business projects only. How can I contact someone who will disable this shit?

**Comment by samus-aran** [comment]
Hi @marcinnformula 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by ghost** [reply]
How to stop this insanity? 
![image](https://github.com/community/community/assets/153630967/887149ec-0722-41dc-acde-f6ce2f7fe83f)

**Reply by ghost** [reply]
How can I ask someone from Github Support? Where is the contact form or email? This is fucking awkward.
